tests/integ/modin/groupby/test_groupby_first_last.py

```diff
@@ -0,0 +1,133 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+import modin.pandas as pd
+import numpy as np
+import pytest
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
+from tests.integ.modin.sql_counter import SqlCounter, sql_count_checker
+from tests.integ.modin.utils import eval_snowpark_pandas_result
+
+data_dictionary = {
```

Comment:
Can you add tests for Series as well?

Reply:
Series is tested through `__getitem__`. We don't yet support grouping by on a series where `by` is another series or collection.